CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required( VERSION 3.0 )
+
+project( cubature )
+
+set( CMAKE_C_FLAGS " -fPIC ${CMAKE_C_FLAGS}" )
+
+find_package( PkgConfig REQUIRED )
+pkg_check_modules( PKG_FFTW REQUIRED fftw3l )
+include_directories( ${PKG_FFTW_INCLUDE_DIRS} )
+
+add_library( cubature SHARED 
+    clencurt_gen.c
```

Comment:
This file is only used to generate clencurt.h, and doesn't need to be linked into the library.